app/controllers/validation_requests_controller.rb

```diff
@@ -10,16 +10,6 @@ def index
 
   def new; end
 
-  def create
-    case params[:validation_request]
-    when "create_document"
-      redirect_to new_planning_application_additional_document_validation_request_path
-    else
-      flash.now[:error] = "You must select a validation request type to proceed."
-      render "new"
-    end
-  end
-
   private
 
   def send_validation_request_email(request)
```

Comment:
I am also wondering whether we can move these methods on [validation_requests](https://github.com/unboxed/bops/blob/main/app/controllers/concerns/validation_requests.rb)